examples/tac_deploy/.env

```diff
@@ -0,0 +1 @@
+PARTICIPANTS_AMOUNT=2
```

Comment:
Let's set this to 100 for cloud deployment

Reply:
done